dagshub/data_engine/client/loaders/tf.py

```diff
@@ -22,14 +22,22 @@ def __init__(self, *args, **kwargs):
 
     def generator(self):
         for idx in range(len(self)):
-            yield self[idx]
+            yield tuple(self[idx])
```

Comment:
I really don't understand much of how TF/PT works and maybe it's time for me to investigate, but why are you changing it to return tuples now?